### PR TITLE
Fix 'set up' typo

### PR DESCRIPTION
**Pull request by christophebedard**
In this case, it should be "set up" and not just "setup"

### CODE REVIEW

File: launch/launch/launch_service.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def __init__(
         :param: argv stored in the context for access by the entities, None results in []
         :param: noninteractive if True (not default), this service will assume it has
             no terminal associated e.g. it is being executed from a non interactive script
-        :param: debug if True (not default), asyncio the logger are seutp for debug
+        :param: debug if True (not default), asyncio the logger are set up for debug
```
Comment (clalancette) [inline launch/launch/launch_service.py]:
```suggestion
        :param: debug if True (not default), asyncio and the logger are set up for debug
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: launch/launch/launch_service.py (hunk 1 above)
Comment (christophebedard) [reply launch/launch/launch_service.py]: good catch :laughing: fixed

**Review by fujitatomoya — APPROVED** [review]

**Comment by ahcorde** [comment]
https://github.com/Mergifyio backport jazzy

**Comment by ahcorde** [comment]
Pulls: ros2/launch#813
Gist: https://gist.githubusercontent.com/ahcorde/f00f288317936d9c6907a5379948edef/raw/094311b4e4807500bdb8c0917ab18411b59bf369/ros2.repos
BUILD args:  --packages-up-to launch --packages-above-and-dependencies launch
TEST args:  --packages-select launch --packages-above launch
ROS Distro: jazzy
Job: ci_launcher
ci_launcher ran: https://ci.ros2.org/job/ci_launcher/14922
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/22234/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/16494/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1756/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/23000/)

**Pull request merged**